monkey/monkey_island/cc/ui/src/services/AuthService.js

```diff
@@ -81,8 +118,8 @@ export default class AuthService {
     })
   };
 
-  _getTokenFromResponse= (response) => {
-    return _.get(response, 'response.user.'+this.TOKEN_NAME_IN_RESPONSE, undefined);
+  _getTokenFromResponse= (tokenName, response) => {
+    return _.get(response, 'response.user.'+tokenName, undefined);
```

Comment:
Instead of lodash use optional chaining (?.):
`return response?.user?.[tokenName];`

You can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)